index.html

```diff
@@ -1,10 +1,16 @@
 <meta charset="utf-8" emacsmode="-*- markdown -*-">
 <link rel="stylesheet" href="style/style.css">
+<link rel="shortcut icon" type="image/png" href="images/logo/openpbr-logo-favicon.png">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface**
+<center>
+   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-glyph.svg" alt="homepage" height="80" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
+   &nbsp;
+   <a href="https://github.com/AcademySoftwareFoundation/OpenPBR"><img src="images/logo/openpbr-logo-text.svg" alt="homepage" height="100" style="vertical-align: middle; position: relative; bottom: 0px"/></a>
+</center>
```

Comment:
As a minor remark, I think we lost a bit of semantic here by replacing the header text with an image, but I don't know what would be the correct way of doing it within Markdeep.

Reply:
Yeah, though I'm not sure it matters that the Markdeep title is missing. I would have preferred to have the subtitle be the Markdeep title, and put the logo above, but Markdeep doesn't seem to allow that.